src/settings/PatronNotices/TokensList/TokensList.test.js

```diff
@@ -0,0 +1,130 @@
+import React from 'react';
+import { isString } from 'lodash';
+import {
+  screen,
+  render,
+  within,
+} from '@testing-library/react';
+
+import '../../../../test/jest/__mock__';
+
+import tokensProps from '../tokens';
+import TokensList from './TokensList';
+
+jest.mock('@folio/stripes-template-editor', () => ({
```

Comment:
i think we should not mock `TokensSection` like this. jest.fn(() => null) will be enough for us in my opinion. then we can check that each `TokensSection` was executed with the correct props. the only nuance for label id check we should inject intl and replace `FormattedMessage` component with `formatMessage` from intl if the component allows us. if not -  just mock something like jest.fn(({header}) => <div>{header}</div>), and check each `TokensSection` execution with expect.objectContaining(props).

Reply:
The `TokensList` component represents list of tokens(checkboxes) broke into sections, and I suppose would be appropriate to test it deeply.

Reply:
Ok, now i got it. But in my mind render of checkboxes the business of `TokensSection`, and now it's mocked. So, do we realy need to test our mock? I think better test `TokensSection` separately. And in our case, since it is a stripes component, it means never.

Reply:
These are thoughtful questions, @Stanislau-Snapok! I see how this test is written to cover `<TokensList>`, but also agree with you that much of it appears to test the mock of `<TokensSection>`. Your are correct, but in the end it does serve to cover `<TokensList>` as well. 🤷  

I think this is less "test is sub-optimal" than "patterns in our codebase may not make testing easy." I really appreciate your comments here. They encourage us all to think about how to write good tests, and how to write easily testable code.